src/components/EmulGlue.cxx

```diff
@@ -104,10 +103,9 @@ void EmulGlue::handle_compile_order(Urho3D::StringHash, Urho3D::VariantMap& even
     if (!m_vehicle_node)
         m_vehicle_node = node_->CreateChild("EmulGlueVehicle");
 
-    // uncomment once we have a vehicle class
-    // m_vehicle = m_vehicle_node->CreateComponent<Vehicle>();
-    // m_vehicle->Init();
-    // setup_attachments(b_data, *veh_config_opt);
+     m_vehicle = m_vehicle_node->CreateComponent<Vehicle>();
+    // m_vehicle->Init(vconf); 
```

Comment:
the `init()` is now changed to `init(const smce::VehicleConfig& vconf)` but i don't know how access the `VehicleConfig` here

Reply:
`*veh_config_opt`